docs/lake/README.md

```diff
@@ -189,10 +189,20 @@ zed lake branch
 ### Use
 
 The `use` command provides a means to set the current branch context
-to the indicate branch, e.g.,
+to the indicated branch, e.g.,
 ```
 zed lake use staging
 ```
+As previously noted, Zed lakes can be used without thinking about branches.
+The `use` command includes a `-p` option that can specify a pool name, which
+sets the current context to the `main` branch of that pool. A user that's not
+yet ready to fully embrace branches could therefore have locked in this
+context by executing the following command after running [`create`](#create)
+to initiate our new example pool.
+```
+zed lake use -p logs
+```
+
```

Comment:
Hopefully it's not too controversial that I mention this here. Part of why I advocated having a branch-oblivious option is because we'd already revealed Zed lakes to many beta users before branches existed and they seemed satisfied with the functionality thus far. Maybe 100% of them will be Git-aware and on-board with the role of branches as soon as they see the expanded doc. However, for those users that are still climbing the gentle slope, I'd like to have this option revealed somewhere obvious so they know they can effectively still do all their work in a pool without having to think about branches until they're ready.